includes/admin/meta-boxes/class-wc-meta-box-product-data.php

```diff
@@ -365,20 +365,20 @@ public static function save( $post_id, $post ) {
 				'backorders'         => isset( $_POST['_backorders'] ) ? wc_clean( wp_unslash( $_POST['_backorders'] ) ) : null,
 				'stock_status'       => wc_clean( wp_unslash( $_POST['_stock_status'] ) ),
 				'stock_quantity'     => $stock,
-				'low_stock_amount'   => isset( $_POST['_low_stock_amount'] ) ? wc_stock_amount( wp_unslash( $_POST['_low_stock_amount'] ) ) : null,
+				'low_stock_amount'   => '' !== $_POST['_low_stock_amount'] ? wc_stock_amount( wp_unslash( $_POST['_low_stock_amount'] ) ) : null,
```

Comment:
This generate the follow PHP notice when stock management is deactivated on WooCommerce:

```
PHP Notice:  Undefined index: _low_stock_amount in wp-content/plugins/woocommerce/includes/admin/meta-boxes/class-wc-meta-box-product-data.php on line 368
```

Also this don't fix the issue, need to set as empty string instead of `null`, see the logic here:

https://github.com/woocommerce/woocommerce/blob/889cf64ac465826104080e5f8ce23e09109cf777/includes/abstracts/abstract-wc-product.php#L982-L990

```
$ wp shell
wp> absint( null )
=> int(0)
```

This should solve all issues:

```php
'low_stock_amount'   => isset( $_POST['_low_stock_amount'] ) && '' !== $_POST['_low_stock_amount'] ? wc_stock_amount( wp_unslash( $_POST['_low_stock_amount'] ) ) : '',
```

Reply:
@claudiosanches Thanks. I've update the PR.